ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import XCTest
+import FluentUI
+
+class ActivityIndicatorTest: XCTestCase {
+    let app = XCUIApplication()
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app.launch()
+
+        if app.staticTexts["FluentUI DEV"].exists {
+            app.staticTexts["ActivityIndicator"].tap()
+        } else if !app.staticTexts["ActivityIndicator"].exists {
+            app.buttons["FluentUI DEV"].tap()
+            app.staticTexts["ActivityIndicator"].tap()
+        }
+    }
+
+    func testSizes() throws {
+        XCTAssertEqual(app.images.element(boundBy: 0).identifier, "Activity Indicator that is In progress of size 4")
```

Comment:
Interesting note: these tests fail if your simulator is set to any language other than `en-US`. Are we okay with that?

Reply:
Oh I didn't even think to check that. Where do we change the language? I tested changing it in Edit scheme -> Run -> Options -> App language and that worked fine.

Reply:
I think any tests we write should eventually work for any language we configure the tests to run on. Accessibility label/accessibility names both get localized, can we build the accessibility identifier off of something else? 

Looks like the accessibility label is created from the isAnimating property on the activity indicator, maybe we can use that directly. Color seems a little more tricky - I wonder if we can use the RGB value